qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
I don't think so, if we do that at bootstrapping process we can pull from a newly created "logstash-latest" url and keep this done only once during slowest part of process, doing that in test execution it unnecessary slows the process. But this are my preference, it does not means is perfect 👍